pom.xml

```diff
@@ -50,11 +51,84 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.csanchez.kubernetes</groupId>
-      <artifactId>kubernetes-java</artifactId>
-      <version>0.0.3</version>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-api</artifactId>
+      <version>2.2.5</version>
+    </dependency>
+
+    <!-- Required to prevent a version conflict with jenkins-core NoSuchMethodError:org.springframework.util.ClassUtils.isCglibProxyClass -->
```

Comment:
I've got strange behavior with dependencies not being packaged so preferred to make it explicit